packages/modelence/src/app/systemConfigSchema.ts

```diff
@@ -0,0 +1,21 @@
+import { ConfigSchema } from "@/config/types";
```

Comment:
Related to the other comment: I don't think we should put all configs here in one place. We already use a modular system, so each module is supposed to define its own config schema. The current problem is that system modules use some configs but they don't seem to properly define them. So the right way to do this would be to either use an existing module (like _system.user) or create a new _system.auth module and pass these into it as a config schema.

Reply:
Example from Typesonic:

```ts
const configSchema: ConfigSchema = {
  'format.maxLineLength': {
    type: 'number',
    default: 40,
    isPublic: true,
  },
};

export default new Module('typewriterText', {
  stores: [dbTexts],
  queries,
  cronJobs,
  configSchema
});

```